app/code/community/Hps/Transit/Model/Payment.php

```diff
@@ -162,13 +162,25 @@ private function _authorize(Varien_Object $payment, $amount, $capture)
             }
 
             $response = $builder->execute();
-
-            if ($response->responseCode !== '00' || $response->responseMessage === 'Partially Approved') {
+            
+            if ($response->responseCode !== '00') {
```

Comment:
I think that the code inside the if statement will never be run because response code different than '00' throw an exception. An exception is thrown in checkResponse function (https://github.com/hps/php-sdk/blob/962f6826c5938ec9b7c13dbad911970c019057da/src/Gateways/TransITConnector.php#L573).

Reply:
@slogsdon shall we update the PHP SDK to allow the AVS failure code and handle this in plugin side